dts/arm/st/l1/stm32l1.dtsi

```diff
@@ -419,6 +419,16 @@
 			status = "disabled";
 			label = "EEPROM_0";
 		};
+
+		dma1: dma@40026000 {
```

Comment:
Please do this in a dedicated commit/PR

Reply:
is it correct if `dma1: dma@40026000 `  is set in the overlay of the **nucleo_152re** for the _test/drivers/spi/spi_loopback_ ?